zh-hans/howtouse/mlsql_lang_cli.md

```diff
@@ -11,14 +11,16 @@
   - [mlsql lang mac](http://download.mlsql.tech/mlsql-lang-darwin-amd64.tar.gz)
   - [mlsql lang linux](http://download.mlsql.tech/mlsql-lang-linux-amd64.tar.gz)
 
-下载后对 tar 包进行解压缩，放置到固定的目录，并配置环境变量（本示例为/opt/mlsql-lang），具体命令如下：
+下载后对 tar 包进行解压缩，放置到固定的目录，并配置环境变量，具体命令如下：
 ```shell
 echo "
-export MLSQL_LANG_HOME=/opt/mlsql-lang
+export MLSQL_LANG_HOME=/opt/mlsql-lang/mlsql-app_2.4-2.1.0-SNAPSHOT
```

Comment:
Is the dir name extracting from `mlsql-lang-linux-amd64.tar.gz` or `mlsql-lang-darwin-amd64.tar.gz`  mlsql-app_2.4-2.1.0-SNAPSHOT?